mne/decoding/tests/test_base.py

```diff
@@ -149,19 +150,19 @@ def inverse_transform(self, X):
         # Retrieve final linear model
         filters = get_coef(clf, 'filters_', False)
         if hasattr(clf, 'steps'):
-            if hasattr(clf.steps[-1][-1].model, 'best_estimator_'):
+            if hasattr(clf.steps[-1][-1].model_, 'best_estimator_'):
                 # Linear Model with GridSearchCV
-                coefs = clf.steps[-1][-1].model.best_estimator_.coef_
+                coefs = clf.steps[-1][-1].model_.best_estimator_.coef_
             else:
                 # Standard Linear Model
-                coefs = clf.steps[-1][-1].model.coef_
+                coefs = clf.steps[-1][-1].model_.coef_
         else:
-            if hasattr(clf.model, 'best_estimator_'):
+            if hasattr(clf.model_, 'best_estimator_'):
                 # Linear Model with GridSearchCV
-                coefs = clf.model.best_estimator_.coef_
+                coefs = clf.model_.best_estimator_.coef_
             else:
                 # Standard Linear Model
-                coefs = clf.model.coef_
+                coefs = clf.model_.coef_
```

Comment:
It might be possible to simplify the above with `getattr` and its default, but not sure that's actualy more readable